commons/README.md

```diff
@@ -14,7 +14,7 @@ make install
 ```
 sudo -u postgres psql -c "CREATE USER \"www-data\" WITH PASSWORD 'www-data';"
 
-DATABASE=c2cgeoportal_tests
+DATABASE=c2cgeoportal_test
```

Comment:
Just one test?

Reply:
To be consistent with admin part, is this really relevant ?

Reply:
no :-)